frida_tools/_magics.py

```diff
@@ -0,0 +1,111 @@
+import abc
```

Comment:
Perhaps we should name this file `_repl_magic` or something? (Plural sounds a bit strange so perhaps singular is better?)